<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthentificationForm</class>
 <widget class="QWidget" name="AuthentificationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedAuthWidget">
     <widget class="QWidget" name="pageLogin">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="logoLoginLabel">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>132</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="usrnameLoginLineedit">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
	qproperty-frame: false;
	margin-bottom: 5px;
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="ClickableLabel" name="registerLabel">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>ori1Uni</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel#registerLabel {
	text-decoration: underline;
}</string>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="passLoginLineedit">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
	qproperty-frame: false;
	margin-bottom: 5px;
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="loginButton">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ori1Uni</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#loginButton {	 
	color: rgb(0, 22, 150);
	background-color: rgb(206, 198, 170);
}</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="authLoginErrorLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ori1Uni</family>
           <pointsize>8</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#authLoginErrorLabel {
	color: rgb(224, 27, 36);
}</string>
         </property>
         <property name="text">
          <string>Error</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRegister">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="ClickableLabel" name="loginLabel">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>ori1Uni</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel#loginLabel {
	text-decoration: underline;
	margin-top: 0px;
}</string>
           </property>
           <property name="text">
            <string>Has an account?</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="usrnameRegisterLineedit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
	qproperty-frame: false;
	margin-bottom: 5px;
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="registerButton">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ori1Uni</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#registerButton {	 
	color: rgb(0, 22, 150);
	background-color: rgb(206, 198, 170);
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="passRegisterLineedit">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
	qproperty-frame: false;
	margin-bottom: 5px;
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="confpassRegisterLineedit">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgba(0, 0, 0, 0%);
	qproperty-frame: false;
	margin-bottom: 5px;
}

QLineEdit:hover {
	border: 1px solid rgb(216, 216, 216);
	border-top: none;
	border-left: none;
	border-right: none;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Confirm password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="logoRegisterLabel">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="authRegisterErrorLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ori1Uni</family>
           <pointsize>8</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#authRegisterErrorLabel{
	color: rgb(224, 27, 36);
}</string>
         </property>
         <property name="text">
          <string>Error</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
